R/wrangle_and_check.R

```diff
@@ -486,17 +486,16 @@ wrangle_results <- function(results_list, sensitivity_analysis_vars) {
   # company trajectories ----------------------------------------------------
   company_trajectories <- results_list$company_trajectories %>%
     dplyr::select(
-      .data$investor_name, .data$portfolio_name, .data$scenario_name, .data$id,
+      .data$investor_name, .data$portfolio_name, .data$scenario_name,
```

Comment:
I think id is now still referred to afterwards, since I get: ```
Adding missing grouping variables: `id` ```

Reply:
the issue was actually that the trajectory data is never ungrouped after the dcf calculation. I added that. Now it passes